src/test/java/com/philvarner/clamavj/test/ClamScanTestCase.java

```diff
@@ -33,15 +39,18 @@ public void testSuccess() throws Exception {
         assertEquals(ScanResult.getPrefix(STREAM_PATH) + OK_SUFFIX, result.getResult());
     }
 
+    @Test
     public void testVirus() throws Exception {
-        InputStream is = new ByteArrayInputStream("X5O!P%@AP[4\\PZX54(P^)7CC)7}$EICAR-STANDARD-ANTIVIRUS-TEST-FILE!$H+H*".getBytes());
+        InputStream is = new ByteArrayInputStream(
+                "X5O!P%@AP[4\\PZX54(P^)7CC)7}$EICAR-STANDARD-ANTIVIRUS-TEST-FILE!$H+H*".getBytes());
```

Comment:
Would you mind storing this forbidding-looking string in a constant?

Reply:
That is a change I pulled from the original repo, so to avoid having more differences between our fork and it in the future I'd like to keep it forbidding and redundant